### PR TITLE
Milking order

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/bronze/usaco-832.mdx [hunk 1]
```diff
@@ -22,6 +22,183 @@ In this problem, there are three cases that we have to consider:
...
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define all(x) x.begin(), x.end()
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:32]: please no

File: solutions/bronze/usaco-832.mdx [hunk 2]
```diff
@@ -22,6 +22,183 @@ In this problem, there are three cases that we have to consider:
...
+
+int main() {
+	freopen("milkorder.in", "r", stdin);
+	freopen("milkorder.out", "w", stdout);
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:36]: could you put this @ the bottom where you actually output stuff?

File: solutions/bronze/usaco-832.mdx [hunk 3]
```diff
@@ -22,6 +22,183 @@ In this problem, there are three cases that we have to consider:
...
+	return 0;
+}
+```
+
+</CPPSection>
+
+<JavaSection>
+
+```java
+
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:121]:
```suggestion
```

File: solutions/bronze/usaco-832.mdx [hunk 4]
```diff
@@ -22,6 +22,183 @@ In this problem, there are three cases that we have to consider:
...
+		}
+	}
+
+	cout << ans << endl;
+	return 0;
+}
+```
+
+</CPPSection>
+
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:117]:
```suggestion
```

File: solutions/bronze/usaco-832.mdx [hunk 5]
```diff
@@ -22,6 +22,183 @@ In this problem, there are three cases that we have to consider:
...
+					}
+				}
+				ans = cows.indexOf(1) + 1;
+			} else {
+				// Case 3
+				int i = n - 1;
+				int pointer = m - 1;
+				while (pointer >= 0) {
+					if (cows.contains(hierarchy.get(pointer))) {
+						// If the cow is in the hierarchy and has already been placed.
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:175]: u sure this is <80?

File: solutions/bronze/usaco-832.mdx [hunk 6]
```diff
@@ -34,49 +211,53 @@ fixed = [list(map(int, data[i + 2].split(' '))) for i in range(k)]
 cows = [0 for i in range(n)]
 
 for i in range(k):
-  	cows[fixed[i][1] - 1] = fixed[i][0]
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-832.mdx:37]: what's w/ the python code changes

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
very bing chilling

**Comment by SansPapyrus683** [comment]
also
image
you didn't tick the boxes mate
which makes me wonder if you legit did test ur code

File: solutions/bronze/usaco-832.mdx (hunk 1 above)
Comment (876pol) [reply solutions/bronze/usaco-832.mdx]: fixed

File: solutions/bronze/usaco-832.mdx (hunk 5 above)
Comment (876pol) [reply solutions/bronze/usaco-832.mdx]: oh oops missed that

File: solutions/bronze/usaco-832.mdx (hunk 6 above)
Comment (876pol) [reply solutions/bronze/usaco-832.mdx:37]: the code didnt match up with the editorial
i switched the block labeled "case 2" with the block labeled "case 3"

**Comment by 876pol** [comment]
oh forgot to tick those -- i have tested my code however

**Comment by SansPapyrus683** [comment]
@ryanchou-dev @envyaims

File: solutions/bronze/usaco-832.mdx [hunk 7]
```diff
@@ -22,6 +22,184 @@ In this problem, there are three cases that we have to consider:
...
+			for (int i = 0; i < n; i++) {
+				if (cows[i] == 0) {
+					ans = i + 1;
+					break;
+				}
+			}
+		}
+	}
+
+	freopen("milkorder.out", "w", stdout);
```
Comment (ryanchou-dev) [inline solutions/bronze/usaco-832.mdx:110]: might wanna move this up js for cleanliness

**Review by ryanchou-dev — APPROVED** [review]
good stuff

File: solutions/bronze/usaco-832.mdx (hunk 7 above)
Comment (SansPapyrus683) [reply solutions/bronze/usaco-832.mdx:110]: no i told him to do that

**Pull request merged**